chatbot.py

```diff
@@ -123,13 +130,8 @@ def message_to_bot(H, clf, learn_response):
     obj = list(obj)
     verb = list(verb)
     adj = list(adj)
-    proper_nouns = set()
-    for t in triples:
-        if t[0][1] == "NNP":
-            proper_nouns.add(t[0][0])
-        if t[2][1] == "NNP":
-            proper_nouns.add(t[2][0])
```

Comment:
Redundant code refactored and moved to a similar loop that was already being executed.